modules/file-system/parallelstore/variables.tf

```diff
@@ -91,3 +91,33 @@ variable "import_destination_path" {
   type        = string
   default     = null
 }
+
+variable "file_stripe" {
+  description = "The parallelstore stripe level for files."
+  type        = string
+  default     = "FILE_STRIPE_LEVEL_UNSPECIFIED"
+  validation {
+    condition = contains([
+      "FILE_STRIPE_LEVEL_UNSPECIFIED",
+      "FILE_STRIPE_LEVEL_MIN",
+      "FILE_STRIPE_LEVEL_BALANCED",
+      "FILE_STRIPE_LEVEL_MAX",
+    ], var.file_stripe)
+    error_message = "var.file_stripe must be set to \"FILE_STRIPE_LEVEL_UNSPECIFIED\", \"FILE_STRIPE_LEVEL_MIN\", \"FILE_STRIPE_LEVEL_BALANCED\",  or \"FILE_STRIPE_LEVEL_MAX\""
+  }
+}
+
+variable "directory_stripe" {
+  description = "The parallelstore stripe level for directories."
+  type        = string
+  default     = "DIRECTORY_STRIPE_LEVEL_UNSPECIFIED"
```

Comment:
Curious, how do we come up with these default value ? Or these are set at downstream level ?

Reply:
Unless I misunderstood what "If not set, FileStripeLevel will default to FILE_STRIPE_LEVEL_BALANCED" meant ? I understood the `_UNSPECIFIED` as an alias?
- https://cloud.google.com/parallelstore/docs/reference/rest/v1/projects.locations.instances#filestripelevel
- https://cloud.google.com/parallelstore/docs/reference/rest/v1/projects.locations.instances#directorystripelevel

Reply:
As per the document, if we don't set it will be default to `FILE_STRIPE_LEVEL_BALANCED`. My question is, do we have a reason to explicitly set default to `DIRECTORY_STRIPE_LEVEL_UNSPECIFIED` ?